src/segment_os.go

```diff
@@ -76,6 +77,7 @@ func (n *osInfo) string() string {
 	case "linux":
 		wsl := n.env.getenv("WSL_DISTRO_NAME")
 		p := n.env.getPlatform()
+		n.OS = p
```

Comment:
I would opt to set this regardless of the platform:

```golang
func (n *osInfo) string() string {
	goos := n.env.getRuntimeGOOS()
	switch goos {
	case windowsPlatform:
		n.OS = windowsPlatform
		return n.props.getString(Windows, "\uE62A")
	case darwinPlatform:
		n.OS = darwinPlatform
		return n.props.getString(MacOS, "\uF179")
	case linuxPlatform:
		wsl := n.env.getenv("WSL_DISTRO_NAME")
		p := n.env.getPlatform()
		if len(wsl) == 0 {
			n.OS = p
			return n.getDistroName(p, "")
		}
		n.OS = wsl
		return fmt.Sprintf("%s%s%s",
			n.props.getString(WSL, "WSL"),
			n.props.getString(WSLSeparator, " - "),
			n.getDistroName(p, wsl))
	default:
		n.OS = goos
		return goos
	}
}
```

This also add the two platforms to `environment.go`:

```golang
	darwinPlatform  = "darwin"
	linuxPlatform   = "linux"
```

Reply:
Good point, I amended the PR accordingly.